clients/unshaded/src/test/java/tachyon/client/file/options/OutStreamOptionsTest.java

```diff
@@ -47,17 +47,16 @@ public void builderTest() {
     OutStreamOptions options =
         new OutStreamOptions.Builder(new TachyonConf())
             .setBlockSizeBytes(blockSize)
-            .setHostname(hostname)
             .setTachyonStorageType(tachyonType)
             .setTTL(ttl)
             .setUnderStorageType(ufsType)
             .build();
 
     Assert.assertEquals(blockSize, options.getBlockSizeBytes());
-    Assert.assertEquals(hostname, options.getHostname());
     Assert.assertEquals(tachyonType, options.getTachyonStorageType());
     Assert.assertEquals(ttl, options.getTTL());
     Assert.assertEquals(ufsType, options.getUnderStorageType());
+    Assert.assertTrue(options.getLocationPolicy() instanceof LocalFirstPolicy);
```

Comment:
The default value should be tested in `defaultsTest` below, using `conf.set(USER_FILE_WRITE_LOCATION_POLICY, _)` to control the default policy, and we should test the setter in this test.